src/app/(with-nav-footer)/activities/_components/BestActivityItem.tsx

```diff
@@ -0,0 +1,40 @@
+import starRating from '@/assets/icons/star-rating.svg';
+import Image from 'next/image';
+import { Activity } from '@/lib/types/activities';
+import Link from 'next/link';
+
+interface BestActivityItemProps {
+  activity: Activity;
+}
+
+export default function BestActivityItem({ activity }: BestActivityItemProps) {
+  return (
+    <div className='relative aspect-[1.03] h-full max-h-[384px] min-h-[186px] w-full max-w-[384px] min-w-[186px] cursor-pointer overflow-hidden rounded-3xl'>
+      <Link href={`/activities/${activity.id}`}>
+        <Image
+          src={activity.bannerImageUrl}
+          fill
+          className='object-cover transition-transform duration-300 ease-in-out will-change-transform hover:scale-105'
+          alt='체험 배너 이미지'
```

Comment:
```suggestion
          alt={activity.title}
```
이렇게 설정해놓으면 이미지 alt값이 데이터에 맞게 표시되니까 이미지에 대한 설명에 좋을 것 같습니당

Reply:
넵 코드 리뷰 확인 후, 반영했습니다 감사합니다!